tools/debug-ui/src/utils.tsx

```diff
@@ -45,8 +45,23 @@ export function formatTraffic(bytes_received: number, bytes_sent: number): React
     );
 }
 
-export function addDebugPortLink(peer_addr: string): ReactElement {
-    return <a href={'http://localhost:3000/' + peer_addr.replace(/:.*/, '/')}>{peer_addr}</a>;
+export function addDebugPortLink(peer_network_addr: string): ReactElement {
```

Comment:
Why do we have something that seems to be basically duplicate of the code above? Is it because one is pure JS whereas this one is typescript?